docs/services.md

```diff
@@ -0,0 +1,101 @@
+
+!!! tip
+    Other services are provided by the [`GSB Library`](https://www.gsb.stanford.edu/library){:target="_blank"}, [`Behavioral Lab`](https://www.gsb.stanford.edu/faculty-research/labs/behavioral-lab){:target="_blank"} and [`Information Technology`](https://stanford.service-now.com/services/){:target="_blank"}.
+
+
+## Research Project Support
+!!! important
+    Research project support is only available to faculty and faculty led projects.
+
+We provide a wide array of research support services, including:
+
+- Data management
+- Code optimization
+- Cloud computing
+- Analytics support
+- Data processing
+- ...and more!
+
+!!! warning
+    We cannot complete most analytics requests in less than two weeks.
+
+Are you a GSB researcher with a specific request for support?  Fill out this [form](https://darcrequest.stanford.edu){:target="_blank"}!
+
+## Research Computing Support
+
+Doing computational or data-intensive research can be challenging.  We provide support for all users of our systems to ensure these systems are working as intended and that uses can reach their goals.  While this support is not as extensive as our [faculty support](/_policies/services/#research-project-support){:target="_blank"}, we are able to provide brief consultations on a number of different topics, including:
+
+* Using the [scheduler](/_user_guide/slurm/){:target="_blank"}
+* Choosing a [storage location](/_user_guide/storage/){:target="_blank"}
+* Parallelizing code
+* Selecting a compute platform
+* Setting up and querying databases
+* Scraping websites
+
+Please [reach out](mailto:gsb_darcresearch@stanford.edu){:target="_blank"} to schedule a consultation or ask a question.
+
+
+##  Onboarding New Data
+There are several ways to onboard data, in this document, we will discuss the two most prevalent procedures: data purchase/gifts and moving data from your previous institution.
+
+!!! important
+    It is essential that you contact the [DARC team](mailto:gsb_darcresearch@stanford.edu){:target="_blank"} or the [GSB Library](https://www.gsb.stanford.edu/library){:target="_blank"} before trying to onboard data yourself.  The DARC Team has vast experience working with various data formats and the Library has both legal (contract negotiation) and curation resources.  By negotiating and onboarding data yourself, you put the School and the University risk.  Data that may seem innocuous can contain Personal Identifiable Information or licensed data.
+
+
+- **Scenario one.** Let’s pretend that you’re an incoming faculty and you have several terabytes of data that you want to bring to Stanford. The DARC Team has this expertise and can deploy several tools from [Globus](/_user_guide/data_transfer/){:target="_blank"} to AWS’s Snowball to get your data loaded into our data ethos.  We will work with the data regardless of security level, including high-risk data.
```

Comment:
I don't understand "data ethos". Maybe we can rephrase like so: "If you're an incoming faculty member with terabytes of data to bring to Stanford, the DARC Team can help. We specialize in using tools like Globus and AWS Snowball to securely transfer your data,  including high-risk datasets, into our data infrastructure."

Reply:
ha ha, sounds like something Alex would write.  done.